taxonomies/ingredients.txt

```diff
@@ -4837,6 +4838,13 @@ vegetarian:en:maybe
 # comment:en:All vegan sites state that industrial produced lactic ferments is vegan. Unclear if exceptions do occur. Still need an authoritative source for this.
 # comment:en:A wide variety of synonyms are used, but they all seem to point to the same bacteria.
 
+<en:fermentation starter
+en:fermentation starter
```

Comment:
<en:fermentation starter
en:fermentation starter

Those 2 lines mean that there is an "en:fermentation started" which is a child of "en:fermentation starter", so it does not work.

what does "закваска" mean ? does it mean "fermentation starter" ? or a more specific kind of fermentation starter?

Reply:
> what does "закваска" mean ? does it mean "fermentation starter" ? or a more specific kind of fermentation starter?

It just means "fermentation starter", its origin is not specified. It could be anything from yeast to abomasum.
https://ru.wikipedia.org/wiki/%D0%97%D0%B0%D0%BA%D0%B2%D0%B0%D1%81%D0%BA%D0%B0

> Those 2 lines mean that there is an "en:fermentation started" which is a child of "en:fermentation starter", so it does not work.

How should I fix it?
Should I just remove the first line?

Reply:
There's an "en:ferment" entry:

# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
#
#                              F E R M E N T S     S E C T I O N
#
# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #

# description:en:FERMENT is an agent, such as an enzyme, bacterium, or fungus, that brings about fermentation.
# comment:en:Ferment is thus more generic than enzyme.

en:ferment


I think "fermentation starter" should just be a synonym of "ferment".

So you can add "fermentation starter" as a synonym in the existing entry

en:ferment, fermentation starter

and add a Russian translation